config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
+  resources :products do
+    resources :bookings, only: [ :new, :create]
+  end
+  resources :bookings, only: [ :edit, :update ]
```

Comment:
Should we add a destroy method for Bookings? This suggestion is that what would happen if the rentee or owner wanted to cancel the Booking. Also, should we add a destroy method for Product as well if the owner wanted to delete his camping gear from his/her listing.